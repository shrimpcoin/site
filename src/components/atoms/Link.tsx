import styled from 'styled-components';

type LinkProps = {
  color: string;
  fadecolor: string;
};

export const Link = styled.a<LinkProps>`
  display: inline;
  text-align: center;
  text-decoration: none;
  font-family: 'Manrope';
  font-weight: 200;
  font-size: 20px;
  // vertical-align: middle;
  padding: 1em 2em;

  transition: all 0.2s ease;
  box-sizing: border-box;

  color: ${(props) => props.color};

  &:hover {
    color: ${(props) => props.fadecolor};
    transition: 0.3s ease-out;
  }
`;
