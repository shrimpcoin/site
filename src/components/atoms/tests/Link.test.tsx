import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from '../Link';

test('Link', () => {
  const component = renderer.create(<Link href="#">Link</Link>);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
