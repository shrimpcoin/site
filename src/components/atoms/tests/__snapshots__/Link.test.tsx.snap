// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link 1`] = `
<a
  className="sc-bdnxRM hJVvQQ"
  href="#"
>
  Link
</a>
`;
